cucumber-jparest/src/test/java/uk/gov/homeoffice/digital/sas/cucumberjparesttestapi/stepdefinitions/PayloadManagerStep.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.homeoffice.digital.sas.cucumberjparesttestapi.stepdefinitions;
```

Comment:
Just wondering about the file location a little. The cucumberjparesttestapi folder was to isolate the code used for running the api whereas this code is related to the tests. What do you think?

Reply:
Good point @stephenkaye-ho . I've re-arranged the folders